### PR TITLE
Add README for creator staking pallet

### PR DESCRIPTION
**Pull request by F3Joule**

### CODE REVIEW

File: pallets/creator-staking/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The Subsocial Creator Staking Pallet is a critical component of the Subsocial content creation ecosystem. It provides features for tracking staking information for creators by era, storing general info about total staking amounts per era, and managing the economic interactions between creators and their supporters.
+
+## Features
+
+- **Staking Information**: The pallet tracks staking information for creators by era, allowing users to stake funds to their favorite creators.
+
+- **Unstaking and Withdrawal**: Users can start the unbonding process, and once the unbonding period has passed, they can withdraw their staked funds.
+
+- **Reward Distribution**: The pallet calculates and distributes rewards for stakeholders and creators on a periodic basis (e.g., per era), incentivizing participation in the ecosystem.
```
Comment (olehmell) [inline pallets/creator-staking/README.md:11]:
```suggestion
- **Reward Distribution**: The pallet calculates and distributes rewards for bakers and creators on a periodic basis (e.g., per era), incentivizing participation in the ecosystem.
```

File: pallets/creator-staking/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- **Lock Periods**: To prevent immediate withdrawals, the pallet manages lock periods. Stakers must wait out an unbonding period after initiating an unstaking operation.
```
Comment (olehmell) [inline pallets/creator-staking/README.md:13]: make sense just combine w/ "Unstaking and Withdrawal"

File: pallets/creator-staking/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- **Events**: The pallet emits events for various lifecycle events, such as staking, unstaking, claiming rewards, and era changes, which can be used for monitoring and analytics.
```
Comment (olehmell) [inline pallets/creator-staking/README.md:15]: I guess useless info

File: pallets/creator-staking/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- **Custom RPCs**: The pallet provides custom RPCs for querying staking information.
```
Comment (olehmell) [inline pallets/creator-staking/README.md:17]: I guess useless info

**Review by olehmell — CHANGES_REQUESTED** [review]

File: pallets/creator-staking/README.md (hunk 2 above)
Comment (Yung-Beef) [reply pallets/creator-staking/README.md:13]: same comment about using both "unbonding" and "unstaking"

File: pallets/creator-staking/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## License
+
+This Subsocial Staking Pallet is open-source software released under the [GNU GPL3 License](LICENSE). Feel free to use, modify, and distribute it as needed.
+
+## Acknowledgements
+
+Special thanks for [Astar Network](https://github.com/AstarNetwork) for their [Pallet dapps-staking](https://github.com/AstarNetwork/astar-frame/tree/polkadot-v0.9.39/frame/dapps-staking).
```
Comment (Yung-Beef) [inline pallets/creator-staking/README.md:25]:
```suggestion
Special thanks to [Astar Network](https://github.com/AstarNetwork) for their [Pallet dapps-staking](https://github.com/AstarNetwork/astar-frame/tree/polkadot-v0.9.39/frame/dapps-staking).
```

**Review by olehmell — APPROVED** [review]

File: pallets/creator-staking/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+This Subsocial Staking Pallet is open-source software released under the [GNU GPL3 License](LICENSE). Feel free to use, modify, and distribute it as needed.
+
+## Acknowledgements
+
+Special thanks to [Astar Network](https://github.com/AstarNetwork) for their [Pallet dapps-staking](https://github.com/AstarNetwork/astar-frame/tree/polkadot-v0.9.39/frame/dapps-staking).
+
+## Support and Contact
+
+If you have any questions or need support, please don't hesitate to [open an issue](https://github.com/dappforce/subsocial-parachain/issues).
```
Comment (olehmell) [inline pallets/creator-staking/README.md:23]:
```suggestion
```

File: pallets/creator-staking/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# Creator Staking Pallet
+
+The Subsocial Creator Staking Pallet is a critical component of the Subsocial content creation ecosystem. It provides features for tracking staking information for creators by era, storing general info about total staking amounts per era, and managing the economic interactions between creators and their supporters.
+
```
Comment (olehmell) [inline pallets/creator-staking/README.md:4]:
```suggestion
The Creator Staking pallet enables SUB token holders to support their favorite content creators, communities, and application builders by staking SUB tokens. Both parties (SUB stakers and creators) earn rewards based on the inflation of the SUB token.

```

File: pallets/creator-staking/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+- **Staking Information**: The pallet tracks staking information for creators by era, allowing users to stake funds to their favorite creators.
+
+- **Unstaking and Withdrawal**: Users can start the unstaking process, and once the unstaking period has passed, they can withdraw their staked funds.
+
+- **Reward Distribution**: The pallet calculates and distributes rewards for backers and creators on a periodic basis (e.g., per era), incentivizing participation in the ecosystem.
+
+## License
+
+This Subsocial Staking Pallet is open-source software released under the [GNU GPL3 License](LICENSE). Feel free to use, modify, and distribute it as needed.
```
Comment (olehmell) [inline pallets/creator-staking/README.md:15]:
```suggestion
This Subsocial Staking Pallet is open-source software released under the [GNU GPL3 License](LICENSE).
```

File: pallets/creator-staking/README.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# Creator Staking Pallet
+
+The Subsocial Creator Staking Pallet is a critical component of the Subsocial content creation ecosystem. It provides features for tracking staking information for creators by era, storing general info about total staking amounts per era, and managing the economic interactions between creators and their supporters.
+
+## Features
+
+- **Staking Information**: The pallet tracks staking information for creators by era, allowing users to stake funds to their favorite creators.
+
+- **Unstaking and Withdrawal**: Users can start the unstaking process, and once the unstaking period has passed, they can withdraw their staked funds.
```
Comment (olehmell) [inline pallets/creator-staking/README.md:9]:
```suggestion
- **Unstaking and Withdrawal**: Backers can start the unstaking process, and once the unstaking period has passed, they can withdraw their staked funds.
```

File: pallets/creator-staking/README.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# Creator Staking Pallet
+
+The Subsocial Creator Staking Pallet is a critical component of the Subsocial content creation ecosystem. It provides features for tracking staking information for creators by era, storing general info about total staking amounts per era, and managing the economic interactions between creators and their supporters.
+
+## Features
+
+- **Staking Information**: The pallet tracks staking information for creators by era, allowing users to stake funds to their favorite creators.
```
Comment (olehmell) [inline pallets/creator-staking/README.md:7]:
```suggestion
- **Staking Information**: The pallet tracks staking information for creators by era, allowing backers to stake funds to their favorite creators.
```

**Review by olehmell — CHANGES_REQUESTED** [review]

**Review by siman — APPROVED** [review]

**Review by olehmell — APPROVED** [review]

**Pull request merged**